scripts/metadata_comparison/metadata_comparison/digester.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import json
+from metadata_comparison.lib import logging, operation_ids
+from metadata_comparison.lib.operation_ids import CallNameSequence, JsonObject, OperationId
+from metadata_comparison.lib.comparison_paths import ComparisonPath
+
+import dateutil.parser
+from typing import AnyStr, Dict
+
+Version = "0.0.1"
+verbose = False
+
+
+def main(args: argparse.Namespace) -> None:
+    for path in args.paths:
+        parent_path = ComparisonPath.create(path)
+
+        workflow_path = parent_path / 'workflow.json'
+        operations_dir_path = parent_path / 'operations'
+
+        digest_parent = parent_path / 'digests' / Version
+        digest_path = digest_parent / 'digest.json'
+
+        if not digest_path.exists() or args.force:
+            digest_parent.mkdir()
```

Comment:
What if `digest_path` doesn't exist, but `digest_parent` exists? Wouldn't it throw exception?

Reply:
`digest_path` is a `ComparisonPath` which turns on "mkdir -p" behavior by default. I agree this is not obvious so I  I'll rename the method to `mkdir_p`.